faf-commons-api/src/main/java/com/faforever/commons/api/dto/MeResult.java

```diff
@@ -28,10 +33,11 @@ public String getId() {
     return userId;
   }
 
+  @Data
   public static class Clan {
-    private Integer id;
-    private Integer membershipId;
-    private String tag;
-    private String name;
+    Integer id;
+    Integer membershipId;
+    String tag;
+    String name;
```

Comment:
why remove the private?

Reply:
whoops missed putting that back when going from value to data and back and forth

Reply:
Should be back now